.changeset/thirty-hairs-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/form-control": minor
```

Comment:
Since other PRs specify patches, it would be better to align with them.

- https://github.com/yamada-ui/yamada-ui/pull/3681/files#diff-c58bb10bf537f2bba552305700c98d522c8b68b738d9da4888477475832868f8R2
- https://github.com/yamada-ui/yamada-ui/pull/3673/files#diff-a8b27a4e23ddd3192fb7366ca8c33dc8cc58b3b1f3bdc73ab65ffd87eda3b270R2

```suggestion
"@yamada-ui/form-control": patch
```

Reply:
fixed [c4ec7cf](https://github.com/yamada-ui/yamada-ui/pull/3672/commits/c4ec7cf1956de49385a28ae421fa437da2d30feb)